### PR TITLE
PIM-5756: Fix empty min and max numbers on attributes import

### PR DESCRIPTION
**Pull request by pierallard**
<3 Thanks for taking the time to contribute! You're awesome! <3

De nada

**Description (for Contributor and Core Developer)**

When you import attributes and set min_number and max_number as "empty" column, these properties are set to 0.
This PR fix it by checking before assign variable if column is empty or not (and fix it too for some other fields)

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | n
| Added Behats                      | y
| Changelog updated                 | TODO
| Review and 2 GTM                  | TODO
| Micro Demo to the PO (Story only) | TODO
| Migration script                  | n
| Tech Doc                          | n

### CODE REVIEW

**Comment by ClementGautier** [comment]
Add a unit test then GTM

File: src/Pim/Component/Connector/ArrayConverter/Flat/AttributeStandardConverter.php [hunk 1]
```diff
@@ -103,11 +103,11 @@ protected function convertFields($field, $booleanFields, $data, $convertedItem)
             case 'number_min':
             case 'number_max':
             case 'max_file_size':
-                $convertedItem[$field] = (float) $data;
+                $convertedItem[$field] = ('' === $data) ? null : (float) $data;
                 break;
             case 'sort_order':
```
Comment (momoss) [inline src/Pim/Component/Connector/ArrayConverter/Flat/AttributeStandardConverter.php]: 'sort_order' cannot be null, look at the DB config, you may return an int value :smiley:

**Comment by momoss** [comment]
GTM

**Comment by pierallard** [comment]
Merged in https://github.com/akeneo/pim-community-dev/pull/4410

**Pull request closed without merging**